content/agent/_index.md

```diff
@@ -18,12 +18,12 @@ aliases:
 ---
 
 <div class="alert alert-info">
-Agent v6 is now available, <a href="https://github.com/DataDog/datadog-agent/blob/master/docs/agent/upgrade.md">upgrade to the newest version</a> to benefit from all new functionality.
+Agent v6 is available. <a href="https://docs.datadoghq.com/agent/faq/upgrade-to-agent-v6">Upgrade to the newest version</a> to benefit from all new functionality.
 </div>
 
 ## What is the Agent?
 
-The Datadog Agent is a piece of software that runs on your hosts. Its job is to faithfully collect events and metrics and bring them to Datadog on your behalf so that you can do something useful with your monitoring and performance data. The Datadog Agent is open source: view the source code on GitHub for [Agent v5][1] and [Agent v6][2]. To see all changes between Agent v5 and v6, consult the [Datadog Agent dedicated changes][3] documentation.
+The Datadog Agent is a piece of software that runs on your hosts. Its job is to faithfully collect events and metrics and bring them to Datadog on your behalf so that you can do something useful with your monitoring and performance data. The Datadog Agent is open source: view the source code on GitHub for [Agent v5][1] and [Agent v6][2].
```

Comment:
I don't think we should mention the source code of Agent v5 at all here. The Agent v6 repo's readme will redirect customers to the v5 repo if needed anyway.